action.yml

```diff
@@ -1,22 +1,45 @@
-name: 'Composite Run Steps Action Template'
+name: 'Deploy On-Prem Web Application'
 
-description: 'Add description here'
+description: 'An action that deploys a web application to an on-prem hosted IIS server.'
 
 inputs:
-  input-1:
-    description: 'Add a description here'
+  server:
+    description: 'The name of the target server'
+    required: true
+  service-account-id:
+    description: 'The service account name'
+    required: true
+  service-account-password:
+    description: 'The service account password'
+    required: true
+  source-zip-file-path:
+    description: 'The path to the zip file that contains the application files'
+    required: true
+  deployment-folder-path:
+    description: 'The path to deploy the application files to'
+    required: true
+  clean-deployment-folder:
+    description: 'Remove files that are not in the source zip file, accepts values of true or false'
+    required: false
+    default: 'false'
+  server-public-key:
+    description: 'Path to remote server public ssl key (local path like c:\folder)'
     required: true
-
-outputs:
-  output-1:
-    description: 'Add description here'
-    value: '${{ steps.step_1.outputs.world }}'
 
 runs:
   using: 'composite'
   steps:
-    - name: Step 1
-      id: step_1
-      shell: bash
+    - name: Action
+      id: action
```

Comment:
You don't have to give this a name or id since it's so generic.

Reply:
👍🏽